patch-package
--- a/node_modules/react-google-invisible-recaptcha/dist/index.js
+++ b/node_modules/react-google-invisible-recaptcha/dist/index.js
@@ -40,7 +40,7 @@ var injectScript = function injectScript(locale) {
 
   var script = document.createElement('script');
   script.id = 'recaptcha';
-  script.src = 'https://www.google.com/recaptcha/api.js?hl=' + locale + '&onload=GoogleRecaptchaLoaded&render=explicit';
+  script.src = 'https://www.recaptcha.net/recaptcha/api.js?hl=' + locale + '&onload=GoogleRecaptchaLoaded&render=explicit';
   script.type = 'text/javascript';
   script.async = true;
   script.defer = true;
